src/components/GuestRow.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from "react";
+import moment from "moment";
+
+const GuestRow = ({ guest, setGuestID }) => {
+  const [highlight, setHighlight] = useState(false);
+
+  const Toggle = () => setHighlight(!highlight);
```

Comment:
Very small nitpick, but i'd choose a different name instead of `Toggle`. 

Firstly, since it's a function, you should name it starting with a lower case. When you use an upper case it looks like it's a component (and it's even highlighted as such).

`toggle` is a great action verb to use, but I think you could make it more descriptive and name it `toggleSelectedRow`. But that's probably just my personal preference.